internal/versions/versions_test.go

```diff
@@ -200,6 +212,32 @@ var _ = Describe("list versions", func() {
 		})
 	})
 
+	Context("GetReleaseVersion", func() {
+		var (
+			ocpVersion string
+			err        error
+		)
+
+		BeforeEach(func() {
+			openshiftVersions = &defaultOpenShiftVersions
+			h = NewHandler(logger, mockRelease, versions, *openshiftVersions, "")
+		})
+
+		It("default", func() {
+			for key := range *openshiftVersions {
+				ocpVersion, err = h.GetReleaseVersion(key)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(ocpVersion).Should(Equal(*(*openshiftVersions)[key].ReleaseVersion))
+			}
+		})
+
```

Comment:
Add test for a missing release_version. I explained how it could happen above.

Reply:
Where?

Reply:
"default" test is for when release version is not provided, and "existing release version" is for kube-api where release version is available.